tex/gregoriotex-main.tex

```diff
@@ -444,6 +444,11 @@
     \gre@debugmsg{ifdim}{ manualinitialwidth = 0pt}%
     \ifdim\gre@space@dimen@manualinitialwidth=0 pt\relax%
       \global\gre@dimen@initialwidth=\wd\gre@box@initial %
+      \ifdim\gre@dimen@initialwidth<\gre@space@dimen@minimalinitialwidth\relax%
+        \gre@style@initial%
```

Comment:
I copied that from the manualinitialwidth code immediately below.  I'm pretty sure it's there to handle relative dimensions like em since space@dimen macros are text.  Therefore, I don't think the style lines should be removed.